internal/services/monitor/monitor_scheduled_query_rules_alert_resource.go

```diff
@@ -92,6 +92,12 @@ func resourceMonitorScheduledQueryRulesAlert() *pluginsdk.Resource {
 				Required:     true,
 				ValidateFunc: azure.ValidateResourceID,
 			},
+			"auto_mitigate": {
```

Comment:
in keeping with booleans elsewhere in the provider can we change this to
```suggestion
			"auto_mitigation_enabled": {
```